src/main/java/de/unistuttgart/iste/gits/media_service/service/MediaService.java

```diff
@@ -22,6 +24,10 @@
 @Slf4j
 public class MediaService {
 
+    public static final String BUCKET_ID = "bucketId";
```

Comment:
No please, that's horrible :D If you really want a constant, it would "Media record with id {} not found" and you to string formatting

Reply:
Sorry I  meant the NOT_FOUND and MEDIA_RECORD_WITH_ID